#Common Configuration Parameters
gem2_left_foot_wrench_topic: "/xbotcore/cogimon/ft/l_leg_ft"  #left foot FT
gem2_right_foot_wrench_topic: "/xbotcore/cogimon/ft/r_leg_ft" #right foot FT
gem2_freq: 200    #min freq of all topics
gem2_lfoot_frame: "l_ankle"   #left foot frame 
gem2_rfoot_frame: "r_ankle"   #right foot frame 

#Unsupervisead Gait Phase Learning Parameters
#Only if use_UL is True
gem2_imu_topic: "/xbotcore/cogimon/imu/imu_link" # base_link (body) IMU
gem2_com_topic: "/SERoW/rel_CoM/pose"  #relative to base_link (body) CoM - computed with kinematics
gem2_plot_results: True #plot the results only works when Dimensionality Reduction is performed
gem2_dim: 2
gem2_gt_comparison: False  #compare to ground-truth, to do so user must supply true friction coefficient mu and ground reaction forces and torques.
gem2_save: True  #Save the Training Results to load in real-time
gem2_dim_reduction: "autoencoders" #autoencoders or pca supported at the current time
gem2_clustering: "gmm" #gmm or kmeans supported at the current time
gem2_train_path: "/Users/ownage/Desktop/gem2/src/gem2_test_cogimon_gem2" #path to training data, data must have same(or approx. same) sampling rate e.g. gem2_freq.